### PR TITLE
Fix Static Analysis Issues for VInput Manager Module

### PR DESCRIPTION
**Pull request by vilasrk**
Tracked-On: OAM-105554

### CODE REVIEW

File: lgInputDevice.cpp [hunk 1]
```diff
@@ -42,6 +42,8 @@ lgInputDevice::lgInputDevice()
     mqId = 0;
     virt = false;
     sourceDev.count = 0;
+    mouseType = 0;
+    fd[MAX_DEV-1] = {0};
```
Comment (JaikrishnaNemallapudi) [inline lgInputDevice.cpp]: check space indentation

File: lgInputDevice.cpp [hunk 2]
```diff
@@ -280,7 +282,7 @@ int lgInputDevice::getMsgQ()
     struct input_event ev = {};
     int n = 1;
     while (n > 0) {
-        n = msgrcv(mqId, &ev, sizeof(struct input_event), 1, IPC_NOWAIT);
+        n = msgrcv(mqId, &ev, sizeof(struct input_event) - sizeof(long), 1, IPC_NOWAIT);
```
Comment (JaikrishnaNemallapudi) [inline lgInputDevice.cpp:286]: Add a comment for modification in size argument

File: sendKey.cpp [hunk 3]
```diff
@@ -76,11 +76,11 @@ static void sendVolumeEvent(int32_t ctrl)
     switch (ctrl) {
     case UP:
         buf.bCtrl = UP;
-        msgsnd(mqId, &buf, sizeof(buf), 0);
+        msgsnd(mqId, &buf, sizeof(buf) - sizeof(long), 0);
         break;
     case DOWN:
         buf.bCtrl = DOWN;
-        msgsnd(mqId, &buf, sizeof(buf), 0);
+        msgsnd(mqId, &buf, sizeof(buf) -sizeof(long), 0);
```
Comment (JaikrishnaNemallapudi) [inline sendKey.cpp]: space indentation

File: lgInputDevice.cpp [hunk 4]
```diff
@@ -292,8 +294,12 @@ static void sendKeyThread(lgInputDevice *vDev)
     lgInputDevice *vD = vDev;
     vD->type = MOUSE;
     int mqId = vD->getMsgQ();
+    if (mqId < 0) {
+            cout << "Failed to get msgq id" << endl;
```
Comment (JaikrishnaNemallapudi) [inline lgInputDevice.cpp]: space indentation

**Review by JaikrishnaNemallapudi — COMMENTED** [review]
have a few comments

**Review by JaikrishnaNemallapudi — APPROVED** [review]
looks good

**Review by JeevakaPrabu — APPROVED** [review]

**Review by JeevakaPrabu — APPROVED** [review]

**Pull request merged**